webanno/webanno
# #1474 - Dynamic resizing string feature

**What's in the PR**
- add dynamic textarea feature editor

**How to test manually**
* Create a new string feature
* Enable multiple rows
* Enable dynamic size, see if collapsed and expanded rows disappears
* Check if the new feature resize in the annotation sidebar when filled

**Automatic testing**

**Documentation**
